src/test.js

```diff
@@ -140,3 +141,13 @@ test('TEST: FAILURE POST/PATCH - Recieve a response with an error when the param
   );
   expect(modifyResponse).toEqual(expect.objectContaining(putMock.failure.response));
 });
+
+test('TEST: SIGNUP FAILURE - Recieve a response with an error when the credentials are wrong', async () => {
+  const response = await firestoreService.signUp(authMock.signUp.email, authMock.signUp.password);
+  expect(response).toEqual(expect.objectContaining(authMock.signUp.response));
+});
+
+test('TEST: LOGIN FAILURE - Recieve a response with an error when the credentials are wrong', async () => {
+  const response = await firestoreService.login(authMock.login.email, authMock.login.password);
+  expect(response).toEqual(expect.objectContaining(authMock.login.response));
+});
```

Comment:
isn't there any way to test when the login/signup is succesfull?

Reply:
I've added a login success, the signup will be quite difficult as we will hace to delete each user after creating it. And it's not very simple to do so